Sources/ViewInspector/SwiftUI/TreeView.swift

```diff
@@ -9,7 +9,7 @@ internal extension ViewType {
 extension ViewType.TreeView: SingleViewContent {
 
     static func child(_ content: Content) throws -> Content {
-        let view = try Inspector.attribute(path: "root|content", value: content.view)
```

Comment:
This is not backward compatible with older os versions, see how this is worked around in a [similar case here](https://github.com/nalexn/ViewInspector/blob/4effbd9143ab797eb60d2f32d4265c844c980946/Sources/ViewInspector/Modifiers/CustomStyleModifiers.swift#L15)

Reply:
Thanks! I added a workaround similar to the one you pointed out. Tests now pass on both iOS 14.5 and 16.2.

One question I had while looking at the `testEnclosedView()` test below: 
**Is there a way to inspect the menu items and find the `Text("Menu")`?**

https://github.com/nalexn/ViewInspector/blob/7a45551ab00f0ff9c8a0996374edcba73705438b/Tests/ViewInspectorTests/SwiftUI/TreeViewTests.swift#L9-L14

Reply:
When I print the `sut` with `Inspector.print`, I can see `"Menu"` string in the structure, which is a good sign - searching / direct inspection should be possible